Frameworks/CoreLocation/CLLocationManager.mm

```diff
@@ -70,7 +70,7 @@ static void _requestExtendedExecutionAsync() {
         } else if (status == WF::AsyncStatus::Canceled) {
             NSTraceError(TAG, @"Extended execution request was canceled");
         } else {
-            NSTraceError(TAG, @"Requesting extended execution failed with error %08x", op.ErrorCode());
+            NSTraceError(TAG, @"Requesting extended execution failed with error %08lx", (unsigned long)op.ErrorCode());
```

Comment:
These casts are in the Apple-recommended Xcode autofixit format for string formats of unknown integer types.